usr/sbin/omv-backup

```diff
@@ -37,7 +36,12 @@ purgeOld()
     if [[ ${keep} -gt 0 ]]; then
         _log "Purging old files..."
         find "${backupDir}" -maxdepth ${OMV_BACKUP_MAX_DEPTH} -type f -mtime +${keep} -name "${OMV_BACKUP_FILE_PREFIX}*" -delete
-        _log "Purging done."
+        if [ $? -eq 0 ]; then
+            _log "Purging done."
+        else
+            _log "ERROR: Purge failed!"
```

Comment:
You can use [omv_error](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/usr/share/openmediavault/scripts/helper-functions#L219) here.